### DISCUSSION TITLE
requires php ^7.2 -> your php version (8.0.2)

### DISCUSSION THREAD

**Discussion by defconcorp** [spatie/db-dumper#144] [Q&A] [answered]
Problem 1
    - geekcom/validator-docs is locked to version 3.5.1 and an update of this package was not requested.
    - geekcom/validator-docs 3.5.1 requires php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.

**Comment by pgrimaud** [comment, marked as answer]
This issue is not related to this package.

Your problem comes from `geekcom/validator-docs` which is not (yet) compatible with PHP 8.0.
One PR is waiting for validation https://github.com/geekcom/validator-docs/pull/108
